egs/wsj/s5/utils/subset_data_dir_tr_cv.sh

```diff
@@ -20,6 +20,8 @@
 # especially for the cases where dataset contains multiple different corpora,
 # where type of speakers or recording channels may be quite different 
 
+echo "$0 This script is deprecated, please use: 'utils/nnet/subset_data_tr_cv.sh'"
+
```

Comment:
If the interfaces are compatible, can you make this a soft link?

Reply:
okay, I'll merge them and create symlink...